Enigmatry.Entry.AspNetCore.Authorization/Requirements/UserHasPermission.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Enigmatry.Entry.AspNetCore.Authorization.Attributes;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Logging;
+
+namespace Enigmatry.Entry.AspNetCore.Authorization.Requirements;
+
+#pragma warning disable CA1711 // Identifiers should not have incorrect suffix
```

Comment:
This warning is legit.

Reply:
In that case we cannot use a name that ends with 'Permission'. Any suggestions?

Reply:
policy.AddRequirements(new UserHasPermission.Requirement()))

Reply:
So you're suggesting to change the class structure to circumvent an arbitrary (imho) naming guideline?

Reply:
No, I'm trying to find sense here before I continue in Chat GPT style ^^

Can't we add something to dummy class or can we get rid of it?

Reply:
Removed nesting.